index.html

```diff
@@ -245,6 +249,10 @@
   git add hello_world.txt
   git commit -m "First commit. Added hello world to repository."
             </code></pre>
+            <p>Do not forget the -m flag, if you do...this will happen.</p>
+            <div class="fragment">
+              <iframe src="http://giphy.com/embed/12KiGLydHEdak8" width="480" height="276" frameBorder="0" style="max-width: 100%" class="giphy-embed" webkitAllowFullScreen mozallowfullscreen allowFullScreen></iframe>
```

Comment:
I didn't even know I submitted a PR for that lol.

On Fri, Apr 10, 2015 at 9:41 AM, Julie Cameron <notifications@github.com>
wrote:

> In index.html
> <https://github.com/gdiannarbor/intro-git-github/pull/1#discussion_r28144702>
> :
>
> > @@ -245,6 +249,10 @@
> >    git add hello_world.txt
> >    git commit -m "First commit. Added hello world to repository."
> >              </code></pre>
> > +            <p>Do not forget the -m flag, if you do...this will happen.</p>
> > +            <div class="fragment">
> > +              <iframe src="http://giphy.com/embed/12KiGLydHEdak8" width="480" height="276" frameBorder="0" style="max-width: 100%" class="giphy-embed" webkitAllowFullScreen mozallowfullscreen allowFullScreen></iframe>
>
> I'm cool with this, but def follow up the [image: :boom:] with a "not
> really, it's just going to launch the command line's default text editor -
> which is probably vim, or nano or something - and make you add your comment
> there - but using the editor has a learning curve and is out of scope for
> this class. so stick with the -m flag, it's easier and what most people
> use anyway."
>
> Just don't want to scare anyone into thinking if they forget two
> characters everything will fail.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdiannarbor/intro-git-github/pull/1/files#r28144702>.
>